docs/content/tutorial/step_03.ngdoc

```diff
@@ -88,6 +88,19 @@ Let's see an example:
     });
 ```
 
+```html
+<html ng-app="myApp">
+<head>
+  <script src="bower_components/angular/angular.js"></script>
+  <script src="app.js"></script>
+</head>
+<body>
+   <!-- The following line is how to use the greetUser angular.js component above in your html doc-->
+  <greet-user></greet-user>
+ </body>
```

Comment:
Inconsistent indentation is the lines above.

Reply:
Resolved, thanks for catching that

Reply:
Ok, finally got this one.  I was looking at the content of the body tag, not the tag itself.  Think I finally got it.